packages/flutter/test/widgets/focus_scope_test.dart

```diff
@@ -1971,6 +1971,46 @@ void main() {
         ),
       );
     });
+
+    testWidgets('Focus gains keyboard focus when it gains accessibility focus', (WidgetTester tester) async {
```

Comment:
nit: is "keyboard focus" the right term? I thought we referred to it as "input focus".